pywbem/cim_operations.py

```diff
@@ -1172,12 +1209,230 @@ def __repr__(self):
                "default_namespace=%r, x509=%s, verify_callback=%r, " \
                "ca_certs=%r, no_verification=%r, timeout=%r, " \
                "use_pull_operations=%r, stats_enabled=%r, recorders=%s)" % \
-               (self.__class__.__name__, self.url, creds_repr, self.conn_id,
-                self.default_namespace, x509_repr, self.verify_callback,
-                self.ca_certs, self.no_verification, self.timeout,
-                self.use_pull_operations, self.stats_enabled,
+               (self.__class__.__name__, self.url, creds_repr,
+                self.conn_id, self.default_namespace, x509_repr,
+                self.verify_callback, self.ca_certs, self.no_verification,
+                self.timeout, self.use_pull_operations, self.stats_enabled,
                 recorder_list)
 
+    @classmethod
+    def configure_logger(cls, name, log_dest=DEFAULT_LOG_DESTINATION,
+                         detail_level=None,
+                         log_filename=DEFAULT_LOG_DESTINATION,
+                         connection=None):
+        """
+        A log configuration method to configure a python logger to
+        log user-issued calls to :class:`~pywbem.WBEMConnection` methods that
+        drive WBEM operations (see :ref:`WBEM operations`) and the
+        responses/exceptions before they are passed back to the user.
+
+        Parameters:
+
+          name (:term:`string`):
+            Simple name (ex. 'api') for the log to be configured or `all` if
+            all of the :class:`~pywbem.WBEMConnection` loggers are to
+            be configured. The value must be one the named defined in
+            :data:`~pywbem._logging.LOGGER_SIMPLE_NAMES`
+
+          log_dest (:term:`string`):
+            String defining the destination for this log or None.
+
+            If it is a :term:`string` defining one of the destinations defined
+            in :data:`~pywbem._logging.LOG_DESTINATIONS` a Python logger is
+            configured for that destination including configuring a logging
+            handler, the message format and setting the log level to DEBUG.
+
+            If `None` a logger with the Python Python NullHandler is configured.
```

Comment:
Looking at the code, that is not the case: If the string "none" is passed, a null handler is configured. If `None' is passed, the logger is not configured.

Reply:
Fixed.  DONE